config/routes.rb

```diff
@@ -14,10 +14,12 @@
 
   namespace :api, defaults: { format: 'json' } do
     scope '(:apiv)', module: :v2, defaults: { apiv: 'v2' }, apiv: /v1|v2/, constraints: ApiConstraints.new(version: 2, default: true) do
-      get 'generic', to: 'disks#generic'
-      constraints(id: %r{[^/]+}) do
-        get 'hosts/:id', to: 'disks#host'
-        get 'subnets/:id', to: 'subnet_disks#subnet'
+      scope :bootdisk do
```

Comment:
The non-api prefix is just disks, not bootdisk. Could we be consistent?

Reply:
One could argue that the leading `bootdisk` with 19.x was a name of the engine, not the resource, but yeah, whatever works.